charts/skypilot/values.yaml

```diff
@@ -100,6 +100,23 @@ ingress:
   # Deprecated: use ingress-nginx.controller.service.nodePorts.https instead
   httpsNodePort: 30051 # Set to null to automatically assign a random port
 
+  # OAuth2 Proxy configuration for authentication
+  oauth2-proxy:
+    # Enable/disable OAuth2 Proxy deployment
+    enabled: false
+    # Okta domain URL for OIDC issuer
+    oidc-issuer-url: "https://your-okta-domain.okta.com"
```

Comment:
Since it is a required field, maybe we can leave this null and validate whether it is set when `oauth2-proxy.enabled=true`